components/fallback/validate.hpp

```diff
@@ -28,6 +28,14 @@ namespace Fallback
         for (std::vector<std::string>::const_iterator it = tokens.begin(); it != tokens.end(); ++it)
         {
             std::string temp = Files::EscapeHashString::processString(*it);
```

Comment:
In fact, I'm pretty sure that because `temp` is an EscapeHashString-processed string, it's impossible for it to contain a hash.

Reply:
Derp.